include/mbedtls/ssl_ciphersuites.h

```diff
@@ -308,7 +308,15 @@ const mbedtls_ssl_ciphersuite_t *mbedtls_ssl_ciphersuite_from_string( const char
 const mbedtls_ssl_ciphersuite_t *mbedtls_ssl_ciphersuite_from_id( int ciphersuite_id );
 
 #if defined(MBEDTLS_PK_C)
+/* Get the weakest (wrt. \c mbedtls_pk_can_do) type of public key
+ * required in the server's certificate for a given cipher suite. 
+ * E.g., an ECDH ciphersuite maps to MBEDTLS_PK_ECKEY_DH. */
```

Comment:
I like that! Thanks for making this change, this is way better that the previous code.

If I understand correctly, the name should be changed next time we have an occasion to rename (semi-)public functions. If so, can we make a note about it somewhere (doesn't have to be in the code).